frontend/ui/widget/opdsbrowser.lua

```diff
@@ -520,6 +525,13 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     end
 end
 
+function OPDSBrowser:createNewDownloadDialog(path, buttons)
+    self.download_dialog = ButtonDialogTitle:new{
+        title = _("Choose format to download into:") .. "\n" .. path,
```

Comment:
Ah, that part was hidden when I looked at it on my phone. "Download format into" is just not proper English though.

Does this have anything to do with the file format (e.g., PDF, EPUB) or is it purely about the path?